src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age && (
+    <p className="Person__age">
+      {`I am ${age}`}
+    </p>
+    )}
+    {isMarried ? (
+      <p className="Person__partner">
+        {`${partnerName} `}
+        is my
+        {sex === 'm' ? (
+          <span> wife</span>
+        ) : (
+          <span> husband</span>
+        )}
```

Comment:
you can just change content of span
```suggestion
          <span>{sex === 'm' ? 'wife' : 'husband'}</span>
```